runtime/integration-tests/src/utils/accounts.rs

```diff
@@ -45,7 +45,17 @@ impl Keyring {
 
 	/// NOTE: Needs to be executed in an externalities environment
 	pub fn id_ecdsa<T: pallet_evm_chain_id::Config>(self) -> AccountId32 {
-		AccountConverter::evm_address_to_account::<T>(self.into())
+		AccountConverter::evm_address_to_account::<T>(self.in_eth())
+	}
+
+	/// Returns the Ethereum address.
+	/// The H160 retrived is NOT the local representation of that account in our
+	/// chain, it's the real address in ethereum.
+	pub fn in_eth(self) -> H160 {
```

Comment:
Nit: Namespace
```suggestion
	pub fn in_evm(self) -> H160 {
```

Reply:
Will change if you confirm [the above](https://github.com/centrifuge/centrifuge-chain/pull/1967#discussion_r1718562357) makes no sense, that probably not 😆